Draft: [IMP] open_academy: Customized form creation T#59063

EmilioSerna opened the pull request:
Added a custom form view in object Course, this displays the Title and the Description of each course.

EmilioSerna committed:
[IMP] open_academy: New menu item T#59063

Added new entries as demo in a new menu item view

EmilioSerna committed:
[IMP] open_academy: New menu item T#59063

Added new entries as demo in a new menu item view

EmilioSerna committed:
[IMP] open_academy: Customized form creation T#59063

Created a custom form view for the Course object, the data displayed is: the title and the description of the course.

EmilioSerna commented:
Please review @andreaarce @frahikLV

andreaarce commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,30 @@
+            </tree>
+        </field>
+    </record>
+    <record id="course_form" model="ir.ui.view">
```
```suggestion
    <record id="course_view_form" model="ir.ui.view">
```

andreaarce commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,30 @@
+        </field>
+    </record>
+    <record id="course_form" model="ir.ui.view">
+        <field name="name">Courses</field>
```
```suggestion
        <field name="name">course.view.form</field>
```

andreaarce commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,30 @@
+    <menuitem id="course_menu" name="Courses" sequence="1"
+    action="course_action"/>
+    <record id="course_tree" model="ir.ui.view">
+        <field name="name">Courses</field>
```
```suggestion
        <field name="name">course.view.tree</field>
```

andreaarce commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,30 @@
+    </record>
+    <menuitem id="course_menu" name="Courses" sequence="1"
+    action="course_action"/>
+    <record id="course_tree" model="ir.ui.view">
```
```suggestion
    <record id="course_view_tree" model="ir.ui.view">
```

EmilioSerna committed:
[IMP] open_academy: Search view creation T#59063

Created a search view for the Course object, making it easier to search based on their title or their description.

EmilioSerna committed:
[IMP] open_academy: Search view creation T#59063

Created a search view for the Course object, making it easier to search based on their title or their description.

frahikLV commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,43 @@
+    </record>
+    <menuitem id="course_menu" name="Courses" sequence="1"
+    action="course_action"/>
+    <record id="course_tree" model="ir.ui.view">
```
```suggestion
    <record id="course_view_tree" model="ir.ui.view">
```

frahikLV commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,43 @@
+            </tree>
+        </field>
+    </record>
+    <record id="course_notebook" model="ir.ui.view">
```
```suggestion
    <record id="course_view_form" model="ir.ui.view">
```

frahikLV commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,43 @@
+        <field name="name">Courses</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <form>
```
Include the `title` in this view too

EmilioSerna replied:
You mean like `<form string="course">`?

frahikLV replied:
No, `title` field of `course` model

frahikLV commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,43 @@
+            </form>
+        </field>
+    </record>
+    <record id="course_search" model="ir.ui.view">
```
```suggestion
    <record id="course_view_search" model="ir.ui.view">
```

frahikLV commented on open_academy/views/course_views.xml:
```diff
@@ -0,0 +1,43 @@
+        <field name="res_model">course</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <menuitem id="course_menu" name="Courses" sequence="1"
```
Adds a line break between all record nodes for readability

frahikLV reviewed (changes requested):
@EmilioSerna Please include these changes

frahikLV reviewed (approved):
@andreaarce LGTM

andreaarce commented:
@EmilioSerna Please attend my suggestions i don't see them
